client/src/components/relatedProducts/relatedProductsList.jsx

```diff
@@ -4,37 +4,142 @@ import styled from 'styled-components';
 import Outfit from './outfitList.jsx'
 import RelationModal from './relationModal.jsx'
 import RelatedCarousel from './relatedCarousel.jsx'
-///
+import axios from 'axios';
+import API_KEY from '../../../../config.js'
+
 const Lists = styled.div`
   margin-left: 23%;
   width:53%;
 `
+const Title = styled.p`
+  text-align: left;
+`
 const RelatedProducts = (props) => {
-  const [isOpen, setIsOpen] = React.useState(false);
-  const product_id = props.mainProduct
+  //Koz has it started as a string
+  const product_id = Number(props.mainProduct)
   const product_style = props.currentStyleId
-    const [modalOpen, setModalOpen] = React.useState(false);
-  console.log('RelatedProducts component called')
+  const [modalOpen, setModalOpen] = React.useState(false);
+  const [accumulatedProductData, setAccumulatedProductData] = React.useState([]);
+  const [productData, setProductData] = React.useState(null);
+  const [productCardClickedOn, setProductCardClickedOn] = React.useState('')
+
+  console.log(`-RELATED PRODUCTS-
+      ID: ${product_id}, Style: ${product_style}`)
+
+  React.useEffect(() => {
+
+    //CALLS FOR MAIN PRODUCT
+    axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${product_id}`, { headers: { Authorization: API_KEY } })
+      .catch(err => { console.log('ERROR IN INFO CALL FOR MAIN PRODUCT: ', err) })
+      .then(response => {
+        let productInfoObj = response.data
+
+        axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${product_id}/styles`, {
+          headers: { Authorization: API_KEY }
+        })
+          .catch(err => { console.log('ERROR IN STYLES CALL FOR MAIN PRODUCT: ', err) })
+          .then(response => {
+            let productStylesObj = response.data;
+
+            axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews/meta?product_id=${product_id}`, {
```

Comment:
I make this Axios request, perhaps we can meet up and discuss how to share data so you don't need to make this request as well.